### PR TITLE
File style update

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Files class style updates.  See #640 

## Type of change

Please delete options that are not relevant.

- Style update

# How Has This Been Tested?

# Checklist:

- [N/A] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: pysat/_instrument.py [hunk 1]
```diff
@@ -58,19 +58,17 @@ class Instrument(object):
         (default=False)
     strict_time_flag : boolean
         If true, pysat will check data to ensure times are unique and
-        monotonically increasing. (default=True)
+        monotonically increasing. (default=True
```
Comment (aburrell) [inline pysat/_instrument.py:61]:
```suggestion
        monotonically increasing. (default=True)
```

**Review by aburrell — COMMENTED** [review]
Found a typo

**Comment by aburrell** [comment]
This branch was significantly behind develop-3, so this is a rather large update.

File: pysat/_files.py [hunk 2]
```diff
@@ -13,59 +19,50 @@
...
-    -----------
-    sat : pysat.Instrument
-        Instrument object
-    manual_org : boolean
-        If True, then pysat will look directly in pysat data directory
-        for data files and will not use default /platform/name/tag/inst_id
-        (default=False)
-    directory_format : string or NoneType
-        directory naming structure in string format. Variables such as
+    Attributes
```
Comment (aburrell) [inline pysat/_files.py:22]: Probably worth going through this docstring and separating parameters and attributes, but this approach was easier when resolving merge conflicts.

File: pysat/_files.py [hunk 3]
```diff
@@ -206,116 +239,172 @@ def __init__(self, sat, manual_org=False, directory_format=None,
         # _load. We start here with the first directory for cases where there
         # are no files.
         self.data_path = self.data_paths[0]
-
-        # Store write to disk preference
+
```
Comment (aburrell) [inline pysat/_files.py:242]:
```suggestion

```

**Comment by aburrell** [comment]
Travis failures are due to changes in upstream branch.

**Review by jklenzing — APPROVED** [review]

**Pull request merged**